src/main/java/com/bullhornsdk/data/model/entity/file/CandidateFileAttachment.java

```diff
@@ -29,6 +29,7 @@
     "fileType",
     "isCopied",
     "isDeleted",
+    "isEncrypted",
```

Comment:
let's add the actual property to the EntityFileAttachment  class was well

Reply:
done